aspnet/fundamentals/configuration/sample/src/UsingOptions/Views/Home/Index.cshtml

```diff
@@ -4,6 +4,8 @@
 <!DOCTYPE html>
 <html>
 <body>
+    <h1>Options</h1>
+    <b>Option1:</b> @Model.Option1
     @Html.DisplayForModel()
```

Comment:
The result of this is something like:

> <h1>Options</h1>
<b>Option1:</b> value1_from_action
<div class="display-label">Option1</div>
<div class="display-field">value1_from_action</div>
<div class="display-label">Option2</div>
<div class="display-field">2</div>

I think you should either use `DisplayForModel()` or custom formatting (for both options), but not both at the same time.